src/cards/target_circle.ts

```diff
@@ -11,13 +11,13 @@ import { CardRarity, probabilityMap } from '../types/commonTypes';
 import { HasSpace } from '../entity/Type';
 
 const id = 'Target Circle';
-const baseRadius = 140;
+const baseRadius = 100;
```

Comment:
Reasoning for the nerf?

Reply:
The radius decrease was done in conjunction with reducing the mana cost. I actually thought this was a pretty nice buff to circle overall, so I'm curious if you'll feel the same after trying it out.

The main reason for these changes is that they're an extension of the cone change. Cone had become better at being a circle than circle was, due to circle's incredibly high base mana cost. So I decreased it, and adjusted the radius accordingly. This means it might be weaker at one single cast, but as you add more you should end up with a bigger circle for the same amount of mana. (And is arguably better at single casting, since it's cheaper and the entire base radius may not be needed)

This brings it more in line with the other targeting spells and should make it a little more viable/versatile/interesting. The goal is to make multi-casting more worthwhile, allow players to use smaller or bigger targeting options as they need to, and make sure circle has a use before super-endgame.

Reply:
Excellent reasoning!